### PR TITLE
Feature/enable or disable login and sign up with metamask

### PR DESCRIPTION
**Pull request by swimsoft**
- Add metamask_login in required env
- Enable/Disable MetaMask button
 - Enable metamask auth label
- Remove metamask_login from env.required
- Scenario for metamask button
- Fix rubocop

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ def ransack_filter_present?(query, name, predicate, value) # rubocop:todo Metric
   def enabled_auth_labels
     options = []
     options << 'Slack' if Comakery::Slack.enabled?
-    options << 'Metamask'
+    options << 'Metamask' if ENV['METAMASK_LOGIN'].present?
```
Comment (Alexey1100) [inline app/helpers/application_helper.rb:39]: Looks like we need to add the variable into `app.json`

Comment (aquabu) [reply app/helpers/application_helper.rb:39]: Yes, it should have a default value of false in app.json and be required.

**Review by Riddlerrr — APPROVED** [review]
👍

File: app.json [hunk 2]
```diff
@@ -75,6 +75,10 @@
       "description": "Discord API client secret",
       "required": false
     },
+    "METAMASK_LOGIN": {
+      "description": "Login and signup with MetaMask",
+      "required": false
+    },
```
Comment (aquabu) [inline app.json:81]: I think this should actually be:
```
   "METAMASK_LOGIN": {
      "description": "Enable login and signup with MetaMask",
      "required": true,
      "value": false
    },
```

**Review by aquabu — CHANGES_REQUESTED** [review]

**Review by Riddlerrr — APPROVED** [review]

**Comment by aquabu** [comment]
@swimsoft Also, there's a merge conflict to be resolved caused by the merge of another app.json PR.

**Comment by Riddlerrr** [comment]
@aquabu Looks like it's ready to merge

**Review by aquabu — APPROVED** [review]

**Pull request merged**